### PR TITLE
[ADF-5562][ADF-5564]Removed !important from from and version list component

### PR DESCRIPTION
**Pull request by arohilaGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

!important is present in adf-aform and versino-list

**What is the new behaviour?**
!important removed from adf-form and version list

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by VitoAlbano — APPROVED** [review]

File: demo-shell/src/app/components/form/form.component.scss [hunk 1]
```diff
@@ -32,7 +32,7 @@
 }
 
 .app-form-config-editor {
-    height: 500px !important;
+    min-height: 500px;
```
Comment (DenysVuika) [inline demo-shell/src/app/components/form/form.component.scss:35]: min-height and height are different properties, and different purpose

Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.scss]: accommodated change with height property.

File: demo-shell/src/app/components/form/form.component.scss [hunk 2]
```diff
@@ -31,8 +31,10 @@
     }
 }
 
-.app-form-config-editor {
-    height: 500px !important;
+.mat-tab-body {
```
Comment (nikita-web-ua) [inline demo-shell/src/app/components/form/form.component.scss:34]: Please do not refer to internal Material classes

Comment (pkunduGL) [reply demo-shell/src/app/components/form/form.component.scss]: Done

File: demo-shell/src/app/components/form/form.component.html [hunk 3]
```diff
@@ -16,13 +16,15 @@ <h3>Error log:</h3>
...
+            <div class="app-form-config-editor">
+                <ngx-monaco-editor
+                style="height: 100%"
                 id="adf-form-config-editor"
-                class="app-form-config-editor"
                 [options]="editorOptions"
                 [(ngModel)]="formConfig"
                 (onInit)="onInitFormEditor($event)">
             </ngx-monaco-editor>
+            </div>
```
Comment (nikita-web-ua) [inline demo-shell/src/app/components/form/form.component.html:27]: please align this div block properly

File: demo-shell/src/app/components/form/form.component.scss [hunk 4]
```diff
@@ -31,9 +31,10 @@
     }
 }
 
+
 .app-form-config-editor {
-    height: 500px !important;
-}
+            height: 500px;
+    }
```
Comment (nikita-web-ua) [inline demo-shell/src/app/components/form/form.component.scss:37]: wrong alignment

File: demo-shell/src/app/components/form/form.component.html (hunk 3 above)
Comment (pkunduGL) [reply demo-shell/src/app/components/form/form.component.html:21]: Done

File: demo-shell/src/app/components/form/form.component.scss (hunk 4 above)
Comment (pkunduGL) [reply demo-shell/src/app/components/form/form.component.scss]: Done

File: demo-shell/src/app/components/form/form.component.html [hunk 5]
```diff
@@ -3,46 +3,36 @@
...
         <mat-tab label="Editor">
-            <ngx-monaco-editor
-                id="adf-form-config-editor"
-                class="app-form-config-editor"
-                [options]="editorOptions"
-                [(ngModel)]="formConfig"
-                (onInit)="onInitFormEditor($event)">
-            </ngx-monaco-editor>
+            <div class="app-form-config-editor">
+                <ngx-monaco-editor style="height: 100%" id="adf-form-config-editor" [options]="editorOptions"
```
Comment (swapnil-verma-gl) [inline demo-shell/src/app/components/form/form.component.html:18]: Can you add this style via a class, or using the id already mentioned here

```
#adf-form-config-editor {
    height: 100%;
}
````

**Comment by swapnil-verma-gl** [comment]
I'm also seeing a lot of formatting changes, especially in the .html file. Can you try to minimise them so as to ensure only the required changes are propagated forward

File: demo-shell/src/app/components/form/form.component.html (hunk 5 above)
Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.html]: Done

File: demo-shell/src/app/components/form/form.component.html [hunk 6]
```diff
@@ -16,13 +16,14 @@ <h3>Error log:</h3>
...
         </mat-tab>
         <mat-tab label="Editor">
-            <ngx-monaco-editor
-                id="adf-form-config-editor"
-                class="app-form-config-editor"
-                [options]="editorOptions"
-                [(ngModel)]="formConfig"
-                (onInit)="onInitFormEditor($event)">
-            </ngx-monaco-editor>
+            <div class="app-form-config-editor">
```
Comment (AleksanderSklorz) [inline demo-shell/src/app/components/form/form.component.html:19]: Any reason you need to wrap ngx-monaco-editor in that extra div? Why you can't have applied class directly to ngx-monaco-editor tag?

File: demo-shell/src/app/components/form/form.component.scss [hunk 7]
```diff
@@ -32,7 +32,10 @@
 }
 
 .app-form-config-editor {
-    height: 500px !important;
+    height: 500px;
+    #adf-form-config-editor {
```
Comment (AleksanderSklorz) [inline demo-shell/src/app/components/form/form.component.scss:36]: can you put empty line before that selector?

Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.scss:36]: added

File: demo-shell/src/app/components/form/form.component.html (hunk 6 above)
Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.html:19]: To increase the specificitiy. The style was not getting added by adding it directly on the editor. 
image
This is the way suggested in their doc.

File: demo-shell/src/app/components/form/form.component.scss (hunk 7 above)
Comment (AleksanderSklorz) [reply demo-shell/src/app/components/form/form.component.scss:36]: I can't see this change, did you push it?

File: demo-shell/src/app/components/form/form.component.html (hunk 6 above)
Comment (AleksanderSklorz) [reply demo-shell/src/app/components/form/form.component.html:19]: I just checked it and that seems really redundant, please see below: 
image


If I apply height: 500px to ngx-monaco-editor directly then it still works like expected. You just need to write proper selector, something like: 

HTML: 
image

SCSS: 
image


then it works like expected without extra div. Looks like their documentation is just wrong.

File: demo-shell/src/app/components/form/form.component.scss (hunk 7 above)
Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.scss:37]: pushed

File: demo-shell/src/app/components/form/form.component.html (hunk 6 above)
Comment (arohilaGL) [reply demo-shell/src/app/components/form/form.component.html]: I have made the changes that you suggested.

**Review by AleksanderSklorz — APPROVED** [review]

**Pull request merged**